src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -467,7 +468,7 @@ static void loc_6C7502(int32_t al);
 static void ride_construction_set_brakes_speed(int32_t brakesSpeed);
 static void ride_construction_tooldown_entrance_exit(const ScreenCoordsXY& screenCoords);
 
-static uint8_t _currentPossibleRideConfigurations[32];
+static track_type_t _currentPossibleRideConfigurations[32];
```

Comment:
These type changes don't need to be part of this PR they can be standalone. Do them in a separate PR. Make this purely for splitting up the booster.

Reply:
They need to be done before the booster _can_ be split from the spinning control. But yeah, should still be a separate PR.